scanpipe/tests/test_models.py

```diff
@@ -625,6 +625,55 @@ def test_scanpipe_codebase_resource_children(self):
         ]
         self.assertEqual(expected, [resource.path for resource in children])
 
+    def test_scanpipe_codebase_resource_walk(self):
+        asgiref_root = self.project_asgiref.codebaseresources.get(path="codebase")
+
+        topdown_paths = list(r.path for r in asgiref_root.walk(topdown=True))
+        expected_topdown_paths = [
+            "codebase/asgiref-3.3.0-py3-none-any.whl",
+            "codebase/asgiref-3.3.0-py3-none-any.whl-extract",
+            "codebase/asgiref-3.3.0-py3-none-any.whl-extract/asgiref",
+            "codebase/asgiref-3.3.0-py3-none-any.whl-extract/asgiref/compatibility.py",
+            "codebase/asgiref-3.3.0-py3-none-any.whl-extract/asgiref/current_thread_executor.py",
```

Comment:
@tdruez The CI checks are failing here because the path strings are over 80 characters in length. What would be the best way to format this? Would it be enough/acceptable to put `# noqa: E501` at the end of the long lines?

Reply:
Let's avoid using `# noqa: E501` as much as possible.

Reply:
The `codebase/asgiref-3.3.0-py3-none-any.whl-extract` does not seems very relevant to the purpose of the test, maybe we can omit that noise in the expected data.